lib/main.js

```diff
@@ -9,6 +9,8 @@ const getPlaylist = module.exports = (linkOrId, optionsOrCallback, callback) =>
     optionsOrCallback = { limit: 100 };
   } else if (typeof optionsOrCallback !== 'object') {
     optionsOrCallback = { limit: 100 };
+  } else if (optionsOrCallback.limit === 0) {
+    optionsOrCallback.limit = Infinity;
   } else if (!optionsOrCallback.limit) {
```

Comment:
Since you're already add it:
Can you also change the `!optionsOrCallback.limit`-Line to sth like `!optionsOrCallback.hasOwnProperty('limit') || isNaN(optionsOrCallback.limit)`?

And test coverage + README entry would be cool